astropy/nddata/nduncertainty.py

```diff
@@ -933,3 +981,10 @@ def _propagate_divide(self, other_uncert, result_data, correlation):
 
     def _data_unit_to_uncertainty_unit(self, value):
         return 1 / value ** 2
+
+    def _convert_to_variance(self):
+        return VarianceUncertainty(self.array ** -1, unit=self.unit ** -1)
```

Comment:
Is there a reason not to use `1 / self.array` here? No need to change this code, I'm just curious if there waws a performance reason or something for writing it this way.

Reply:
Same issue as https://github.com/astropy/astropy/pull/12057/files#r854225811 when `self.unit is None`

Reply:
Fixed. I think the `array ** -1` was a habit I picked up when doing odd things with numpy dtypes in order to conserve the type, but I'm not sure if that's still needed.